orchagent/cbforch.h

```diff
@@ -0,0 +1,50 @@
+#ifndef SWSS_CBFORCH_H
+#define SWSS_CBFORCH_H
+
+#include "orch.h"
+
+using namespace std;
+using namespace swss;
+
+/* Class to handle QoS map tasks. */
+class MapHandler : public unordered_map<string, sai_object_id_t>
+{
+public:
+    /* The map type handled by this handler. */
+    enum Type
+    {
+        DSCP,
+        EXP
+    };
```

Comment:
Warning: This enum declaration gives the members _global_ naming scope which causes all sorts of problems.

The recommended way for doing enums in modern C++ is to declare them as `enum class` rather than `enum` - that way they get qualified as members of a specific enumeration group instead of sharing a global namespace.

So here, if you declare this as `enum class Type`, you would then later refer to them in the code as `MapHandler::Type::DSCP` (or just `Type::DSCP` from within this class's methods).